integration/fuse/src/test/java/alluxio/fuse/AlluxioFuseFileSystemTest.java

```diff
@@ -214,8 +214,10 @@ public void getattrWhenWriting() throws Exception {
     AlluxioURI expectedPath = BASE_EXPECTED_URI.join(path);
 
     FileOutStream fos = mock(FileOutStream.class);
-    when(mFileSystem.createFile(expectedPath)).thenReturn(fos);
+    when(mFileSystem.createFile(any(AlluxioURI.class),
+        any(CreateFileOptions.class))).thenReturn(fos);
```

Comment:
Why do we need the createfileoptions argument?

Reply:
Yeah, because the create file in `AlluxioFuseFileSystem.create()` now will call `FileSystem.createFile(path, options)`.

Reply:
oh I see, I thought the other createFiles did not have the arg but it seems like they are all updated.